misc/contrib/docker/build/confd/templates/local_settings.tmpl

```diff
@@ -0,0 +1,27 @@
+from nsupdate.settings.{{getenv "BUILD"}} import *
+
+STATIC_ROOT='/static'
+MEDIA_ROOT='/upload'
+
+SECRET_KEY = '{{ getenv "SECRET_KEY" }}'
+BASEDOMAIN = '{{ getenv "BASEDOMAIN" }}'
+WWW_HOST='{{ getenv "BASEDOMAIN" }}'
+
+REGISTRATION_OPEN = False
+
+SERVICE_CONTACT = '{{ getenv "SERVICE_CONTACT" }}'
+
+ALLOWED_HOSTS=['localhost', '127.0.0.1', '[::1]', '{{ getenv "BASEDOMAIN" }}']
+
+{{ if getenv "DB_NAME"}}
+DATABASES = {
+    'default': {
+        'ENGINE': 'django.db.backends.postgresql_psycopg2',
```

Comment:
seems like this is just 'ENGINE': 'django.db.backends.postgresql' now.

Reply:
Thanks, I'll update that.